### ISSUE TITLE
UI is resized when using this package

### ISSUE THREAD

**Issue by Vadorequest** [abogushov/django-admin-json-editor#25]
I experienced a visual bug as soon as I started using this package, this is what my UI look like without it:

![greenshot 2019-03-07 11 18 10](https://user-images.githubusercontent.com/3807458/53949891-4a43b200-40cb-11e9-8c6b-91fed5d37b4e.png)

And now, after using the `JSONEditorWidget` in my `admin.py`
![image](https://user-images.githubusercontent.com/3807458/53949984-75c69c80-40cb-11e9-99a7-420e9add88fa.png)

Basically, the UI looks resized, smaller, as if I had zoomed out. Probably a bad CSS rule applying undesired styles on the whole page instead of a sub element or something.

Using Python 3.6, Django 2.1.7

**Comment by FadedCoder** [comment]
Did you ever find a fix for this?

**Comment by Vadorequest** [comment]
I never looked into it, and I'm not using this plugin anymore so I won't.

**Comment by xrmx** [comment]
This should be fixed in master

**Comment by xrmx** [comment]
And now in 0.2.2, closing.
